repo-agent/examples/agent-sandbox-repowatch.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         Issue Body: "{{.Body}}"
         HTML URL: "{{.HTMLURL}}"
   - name: fixes
-    maxActiveSandboxes: 4
+    maxActiveSandboxes: 1
     devcontainerConfigRef: go-devcontainer-json
     pushEnabled: true
```

Comment:
Q: Do we also want to comment out `pushEnabled` here (similar to repo-agent/examples/kro-repowatch.yaml)?